Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
Just did the xcode-select --install

And now 1.5.8 builds I'll test the 1.5.9 without the extrea dependency

On Mon, Jul 18, 2016 at 1:54 PM Dominyk Tiller <notifications@github.com>
wrote:

> In Formula/unbound.rb
> <https://github.com/Homebrew/homebrew-core/pull/3080#discussion_r71184911>
> :
>
> > @@ -13,11 +13,13 @@ class Unbound < Formula
> >
> >    depends_on "openssl"
> >    depends_on "libevent"
> > +  depends_on "expat"
>
> Ah, I'm almost certain it's because you don't have the Command Line Tools
> package installed, which is perfectly valid & supported by both us & Apple,
> but some upstreams struggle with it. Can you remove the expat dependency
> & try this locally:
>
> def install
>   args = %W[    --prefix=#{prefix}    --sysconfdir=#{etc}    --with-libevent=#{Formula["libevent"].opt_prefix}    --with-ssl=#{Formula["openssl"].opt_prefix}  ]
>   args << "--with-libexpat=#{MacOS.sdk_path}/usr/include" unless MacOS::CLT.installed?
>
>   system "./configure", *args
>   inreplace "doc/example.conf", 'username: "unbound"', 'username: "@@HOMEBREW-UNBOUND-USER@@"'
>   system "make", "install"end
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/homebrew-core/pull/3080/files/bae0c807e72cb3fde2dff0a9fabb6c40486b6982#r71184911>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATkHVe_jV1tHZXL2MUpG1NASaDKx4p5xks5qW6-3gaJpZM4JOknY>
> .
>